packages/material-ui/src/Popper/Popper.d.ts

```diff
@@ -1,27 +1,36 @@
 import * as React from 'react';
 import { PortalProps } from '../Portal';
+import { TransitionProps } from '../transitions/transition';
+
+export type PopperPlacementType =
+  | 'bottom-end'
+  | 'bottom-start'
+  | 'bottom'
+  | 'left-end'
+  | 'left-start'
+  | 'left'
+  | 'right-end'
+  | 'right-start'
+  | 'right'
+  | 'top-end'
+  | 'top-start'
+  | 'top';
 
 export interface PopperProps extends React.HTMLAttributes<HTMLDivElement> {
-  anchorEl?: HTMLElement | ((element: HTMLElement) => HTMLElement);
-  children: () => React.ReactElement<any> | React.ReactElement<any>;
+  transition?: boolean;
+  anchorEl?: null | HTMLElement | ((element: HTMLElement) => HTMLElement);
```

Comment:
How is `?` not enough?

Reply:
With `strictNullChecks` or `strict` mode turned on, `?` only allows for passing `undefined`, but passing `null` does work took.

In reality, any developer using `strict` should be capable of figuring out that they can just use `undefined` but documentation example was using `null` so dunno thought I'd add it.

Not really important.